samples/Pdf/21a_Pdf.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use PhpOffice\PhpSpreadsheet\Worksheet\PageSetup;
+use PhpOffice\PhpSpreadsheet\Writer\Pdf\Dompdf;
+use PhpOffice\PhpSpreadsheet\Writer\Pdf\Mpdf;
+use PhpOffice\PhpSpreadsheet\Writer\Pdf\Tcpdf;
+
+require __DIR__ . '/../Header.php';
+$spreadsheet = require __DIR__ . '/../templates/sampleSpreadsheet.php';
+
+$helper->log('Hide grid lines');
+$spreadsheet->getActiveSheet()->setShowGridLines(false);
+
+$helper->log('Set orientation to landscape');
+$spreadsheet->getActiveSheet()->getPageSetup()->setOrientation(PageSetup::ORIENTATION_LANDSCAPE);
+
+function yellowBody(string $html): string
+{
+    $newstyle = <<<EOF
+<style type='text/css'>
+body {
+background-color: yellow;
+}
+</style>
+
+EOF;
+
+    return preg_replace('@</head>@', "$newstyle</head>", $html);
```

Comment:
Let's stick to either `~` or `/` for regexp delimiters.

```suggestion
    return preg_replace('~</head>~', "$newstyle</head>", $html);
```